docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       retries: 3
       start_period: 1m
     volumes:
-      - ./.config:/opt/ceremonyclient/node/.config
+      - ./.config:/root/.config
```

Comment:
will this cause a problem for anyone currently running this?

Reply:
they will not see the the `.config` folder on the host, the container will run fine, so they cannot easily backup `config.yml` and `keys.yml`

Reply:
Thanks for fixing this.